test/jobs/benchmark_regression_job_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+class BenchmarkRegressionJobTest < ActiveJob::TestCase
+  setup do
+    @array = [323, 11, 555, 666, 234, 21, 666, 343, 1, 2]
+    benchmark_result_type = create(:benchmark_result_type)
+    benchmark_type = create(:benchmark_type)
+    repo = create(:repo)
+    10.times do
+      commit = create(:commit, repo: repo)
+      @benchmark_run = create(:commit_benchmark_run,
+        benchmark_result_type: benchmark_result_type,
+        benchmark_type: benchmark_type,
+        initiator: commit
+      )
+    end
+  end
+
+  test "calculates average" do
+    assert_equal 282.2, BenchmarkRegressionJob.new.average(@array)
+  end
+
+  test "calculates standard_deviation" do
+    assert_equal 274.08, BenchmarkRegressionJob.new.standard_deviation(@array).round(2)
+  end
+
+  test "#perform benchmark regression job" do
+    assert_equal @benchmark_run.result.keys , BenchmarkRegressionJob.new.perform(@benchmark_run.id)
+  end
+
+  test "create issue for benchmark regression" do
+    VCR.use_cassette('benChmark_regression') do
```

Comment:
It's just any random cassette name given by me for testing. I will change it to `Benchmark_Regression` once we set the correct api. :smile: